### PR TITLE
moving http 2xx metric report frequency from 200 ms to 30 seconds

### PR DESCRIPTION
**Pull request by danbills**
Reduce frequency of HTTP 2xx reports to Elastic Search.

 @timperrett Could you please review?

### CODE REVIEW

File: elastic/src/main/scala/metrics.scala [hunk 1]
```diff
@@ -17,13 +17,18 @@
...
 
 object metrics {
-  val NonHttpErrors    = counter("elastic/errors/other")
-  val HttpResponse5xx  = counter("elastic/http/5xx")
-  val HttpResponse4xx  = counter("elastic/http/4xx")
-  val HttpResponse2xx  = lapTimer("elastic/http/2xx")
-  val BufferDropped    = counter("elastic/queue/dropped")
-  val BufferUsed       = numericGauge("elastic/buffer/pending", 0)
+  //report these metrics every 30 seconds
+  val i = new Instruments(bufferTime = 30 seconds)
```
Comment (timperrett) [inline elastic/src/main/scala/metrics.scala]: Interesting. What's the thinking here?

Comment (danbills) [inline elastic/src/main/scala/metrics.scala]: The default report timing is 200 ms.  This was effectively producing a constant stream of POSTs to Elastic Search, as every 2xx would update the metric in sort of a perpetual motion machine.

Comment (danbills) [inline elastic/src/main/scala/metrics.scala]: I suppose this should be configurable, not hard-coded.

Comment (timperrett) [inline elastic/src/main/scala/metrics.scala]: I'm ok for now if we hardcode it. Interesting behavior though!

**Pull request merged**